webpack.config.js

```diff
@@ -0,0 +1,51 @@
+var webpack = require('webpack');
+
+var config = {
+  entry: './js/index.js',
+  output: {
+    filename: 'dist/forge.js',
+    library: 'Forge',
+    libraryTarget: 'umd'
+  },
+  externals: {
+    // Don't bundle the 'jquery' package in forge.js, but
+    // instead load from `jQuery` global variable or AMD/CJS package.
+    'jquery': {
+      root: 'jQuery',
+      commonjs2: 'jquery',
+      amd: 'jquery'
+    }
+  },
+  module: {
+    loaders: [
+      { test: /\.js$/, exclude: /node_modules/, loader: 'babel-loader'}
+    ]
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        'NODE_ENV': process.env.NODE_ENV
```

Comment:
It's a pretty standard way of setting debug vs. production environments in the Node world. React, for example, uses this to conditionally switch on and off debugging features in their builds so it seems to make sense for us to do the same. :)